composer.json

```diff
@@ -26,14 +26,14 @@
         "php": ">=5.5.0",
         "helhum/typo3-console-plugin": "^1.5.0",
 
-        "typo3/cms-backend": "^7.6 || ~8.3.0",
-        "typo3/cms-core": "^7.6 || ~8.3.0",
-        "typo3/cms-extbase": "^7.6 || ~8.3.0",
-        "typo3/cms-extensionmanager": "^7.6 || ~8.3.0",
-        "typo3/cms-fluid": "^7.6 || ~8.3.0",
-        "typo3/cms-install": "^7.6 || ~8.3.0",
-        "typo3/cms-saltedpasswords": "^7.6 || ~8.3.0",
-        "typo3/cms-scheduler": "^7.6 || ~8.3.0",
+        "typo3/cms-backend": "^7.6 || ~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
```

Comment:
How about `^8.0 < 8.4` instead? Shorter and easier to change for the coming releases.